Source/FolioReaderContainer.swift

```diff
@@ -69,7 +69,7 @@ open class FolioReaderContainer: UIViewController {
             kCurrentHighlightStyle: 0,
             kCurrentTOCMenu: 0,
             kCurrentMediaOverlayStyle: MediaOverlayStyle.default.rawValue,
-            kCurrentScrollDirection: FolioReaderScrollDirection.vertical.rawValue
+            kCurrentScrollDirection: FolioReaderScrollDirection.defaultVertical.rawValue
```

Comment:
This makes sure the initial value of the scroll direction is the new `.defaultVertical`, and not `.vertical`. This default value works as `vertical` if the user doesn't sets another value for the `config`'s scroll direction.